### PR TITLE
fmc_basic_peripheral: implementation of basic peripheral board

### PR DESCRIPTION
**Pull request by vowstar**
Implementation of basic peripheral board

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "library"]
+	path = fmc_basic_peripheral/library
+	url = git@github.com:vowstar/vowstar_kicad_library.git
```
Comment (sequencer) [inline .gitmodules]: I prefer add a new repo named rocket-pcblib

**Comment by vowstar** [comment]
3D rendering view:
photo_2023-05-17_14-04-44
photo_2023-05-17_14-04-36

**Comment by vowstar** [comment]
Real PCB board:
photo_2023-05-17_14-09-28
photo_2023-05-17_14-06-01
photo_2023-05-17_14-10-14
photo_2023-05-17_14-10-09

**Comment by vowstar** [comment]
This FMC has undergone several bug fixes and PCB proofing, and the basic functions have been verified on vcu118.

photo_2023-05-17_14-02-27

File: .gitmodules (hunk 1 above)
Comment (sequencer) [reply .gitmodules]: github.com/chipsalliance/rocket-pcblib

Comment (vowstar) [reply .gitmodules]: > github.com/chipsalliance/rocket-pcblib

Done

**Pull request merged**